canonical-websites/cn.ubuntu.com
# Add autoprefixer to build toolchain

## Done

Add autoprefixer to build toolchain to fix external link icons

## QA

- Run `./run serve --watch`
- View home page at http://0.0.0.0:8010/
- Verify external link icons display correctly
